test/test_helper.exs

```diff
@@ -2,6 +2,9 @@
 # Needs to be called before ExUnit.start
 {:ok, _} = Application.ensure_all_started(:ex_machina)
 
+# By default, exclude S3 tests
+# To run S3 tests, run `mix test --include s3`
+ExUnit.configure exclude: [:s3]
```

Comment:
Understandable. I can make it included by default. I'll make sure to point that out in the README as well.